downstream/modules/platform/con-controller-provisioning-callbacks.adoc

```diff
@@ -0,0 +1,19 @@
+[id="controller-provisioning-callbacks"]
+
+= Provisioning Callbacks
+
+Provisioning Callbacks are a feature of {ControllerName} that enable a host to initiate a playbook run against itself, rather than waiting for a user to launch a job to manage the host from the {ControllerName} console.
+
+Provisioning Callbacks are only used to run playbooks on the calling host and are meant for cloud bursting.
```

Comment:
is there an explanation of Cloud bursting? (Why does this make me think of Kate Bush?)

Reply:
Added definition "Cloud bursting is a cloud computing configuration that enables a private cloud to access public cloud resources by “bursting” into a public cloud when computing demand spikes."